res/Sectors/M1105/Porlock.txt

```diff
@@ -0,0 +1,446 @@
+# Generated by https://travellermap.com
```

Comment:
The Hive Federation should have 7-9 subsector capitals in Porlock, they currently have 2